legacy-fabric-registry-sync-api-v1/build.gradle

```
@@ -0,0 +1,5 @@
+version = getSubprojectVersion(project, "1.0.0")
+
+dependencies {
+	implementation project(path: ':legacy-fabric-api-base', configuration: 'dev')
```

```suggestion
	implementation project(path: ':legacy-fabric-api-base', configuration: 'dev')
	implementation project(path: ':legacy-fabric-networking-api-v1', configuration: 'dev')
```
This should fix the build I guess